cmd/plugin-overview/plugin-overview.go

```diff
@@ -0,0 +1,109 @@
+// Copyright 2019 The Kubernetes Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// plugin-overview reads the manifests in a directory and creates a markdown overview page
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"regexp"
+	"strings"
+
+	"github.com/golang/glog"
+	"github.com/spf13/pflag"
+
+	"sigs.k8s.io/krew/pkg/index"
+	"sigs.k8s.io/krew/pkg/index/indexscanner"
+)
+
+const (
+	separator  = " | "
+	pageHeader = `# Plugin overview
+
+Below is a table of plugins available in the central krew-index.
+
+`
+)
+
+var (
+	githubRepoPattern = regexp.MustCompile(`.*github.com/([^/]+/[^/#]+)`)
+)
+
+func main() {
+	pluginsDir := pflag.String("plugins-dir", "", "The directory containing the plugin manifests")
```

Comment:
do you get much out of pflag vs stdlib pkg/flags here?

Reply:
No. This is a leftover from when this tool was not its own module.
 Only the usage output looks a bit weird when using vanilla `flags`, though.

Reply:
I wouldn't worry about usage as it's not a primary tool.

Reply:
I see your point now. `glog` hijacks the flags smh.